src/emm/emm_cli.py

```diff
@@ -211,5 +219,24 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100))
+@click.option("--commit-message", default=None, help="Commit message to apply.")
+@click.pass_context
+def git_commit(ctx: click.Context, commit_message: str) -> None:
+    """
+    Perform git commit and add git tracked changes to all modules.
```

Comment:
This reads a little awkwardly. Maybe something like "Create a git commit of changes in each module". Then if there are any missing details, you can include them in the following paragraphs.